SMARTAGRIFOOD/vineyardData/examples/example-normalized.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
there is no id in this payload, this is not a valid one 
the object AgriVineyardBlock with only one subproperty is a bit weird
'subproperties' ?? Why use these name for creating a two levels object